recipe/meta.yaml

```diff
@@ -26,11 +26,14 @@ requirements:
     - numpy >=1.11.0
     - scipy >=0.12.0
     - matplotlib >=1.5.0
-    - theano >=0.8.2
+    - theano >=0.9
     - pandas >=0.18.0
     - patsy >=0.4.0
     - joblib >=0.9
+    - six >=1.10.0
     - tqdm >=4.8.4
+    - h5py >=2.7.0
+    - enum34 >=1.1.6
```

Comment:
I believe there is no `enum34` for `Python 3.6`, right? Actually shouldn't that be only for `Python <3.4`?

If so this would fix it: `enum34 >=1.1.6  # [py<34]`